src/tests/builders/safeTx.ts

```diff
@@ -65,3 +74,40 @@ export function safeSignatureBuilder(): IBuilder<SafeSignature> {
     data: faker.string.hexadecimal({ length: faker.number.int({ max: 500 }) }),
   })
 }
+
+export function safeTxSummaryBuilder(): IBuilder<TransactionSummary> {
+  return Builder.new<TransactionSummary>().with({
+    id: `multisig_${faker.string.hexadecimal({ length: 40 })}_${faker.string.hexadecimal({ length: 64 })}`,
+    executionInfo: executionInfoBuilder().build(),
+    txInfo: txInfoBuilder().build(),
+    txStatus: faker.helpers.enumValue(TransactionStatus),
+  })
+}
+
+export function executionInfoBuilder(): IBuilder<MultisigExecutionInfo> {
+  const num1 = faker.number.int({ min: 1, max: 10 })
+  const num2 = faker.number.int({ min: 1, max: 10 })
+
+  return Builder.new<MultisigExecutionInfo>().with({
+    nonce: faker.number.int(),
+    type: DetailedExecutionInfoType.MULTISIG,
+    confirmationsRequired: Math.max(num1, num2),
+    confirmationsSubmitted: Math.min(num1, num2),
+    missingSigners: Array.from(new Array(Math.min(num1, num2))).map(() => ({
+      value: faker.finance.ethereumAddress(),
+    })),
```

Comment:
You can achieve this within `Array.from`:

```suggestion
    missingSigners: Array.from({ length: Math.min(num1, num2) }, () => ({
      value: faker.finance.ethereumAddress(),
    })),
```